src/i18n/messages/en.js

```diff
@@ -0,0 +1,36 @@
+export default {
+    aor: {
+        action: {
+            delete: 'Delete',
+            show: 'Show',
+            list: 'List',
+            save: 'Save',
+            create: 'Create',
+            edit: 'Edit',
+            cancel: 'Cancel',
+            refresh: 'Refresh',
+            add_filter: 'Add filter',
+            delete_item: 'Delete %{name}',
+            create_item: 'Create %{name}',
+            list_item: '%{name} List',
+        },
+        message: {
+            yes: 'Yes',
+            no: 'No',
+            are_you_sure: 'Are you sure ?',
+            page_out_of_boundaries: 'Page number %{page} out of boundaries',
```

Comment:
You'd better group pagination related labels under a new key, or the `message` key will contain too many unrelated things.